bin/brew

```diff
@@ -86,9 +86,9 @@ then
   export HOMEBREW_EDITOR="$VISUAL"
 fi
 
-# Set CI variable for Azure Pipelines and Jenkins
-# (Set by default on Circle and Travis CI)
-if [[ -z "$CI" ]] && [[ -n "$TF_BUILD" || -n "$JENKINS_HOME" ]]
+# Set CI variable for Azure Pipelines
+# (Set by default on GitHub Actions)
+if [[ -z "$CI" ]] && [[ -n "$TF_BUILD" ]]
```

Comment:
Sorry, I think this should stay as-is. We want to set this still on Jenkins (despite Homebrew not using it). If there's a change here it'd just be adding `Set by default on GitHub Actions, Circle CI and Travis CI` or similar.

Reply:
I understood I will reset and add `GitHub Actions` to this explanation